pkg/services/ngalert/sender/sender_test.go

```diff
@@ -48,7 +49,8 @@ func TestSanitizeLabelName(t *testing.T) {
 	}
 
 	for _, tc := range cases {
-		am := NewExternalAlertmanagerSender()
+		logger := log.New("ngalert.sender.external-alertmanager")
```

Comment:
`log.NewNopLogger()` exists if you want extra safety. no idea if it matters, or if the grafana logger will fill up because it's not getting flushed in a test context or something.